test/test_transforms.py

```diff
@@ -410,6 +410,25 @@ def test_batch_MelScale(self):
         self.assertTrue(computed.shape == expected.shape, (computed.shape, expected.shape))
         self.assertTrue(torch.allclose(computed, expected))
 
+    def test_batch_InverseMelScale(self):
+        n_fft = 8
+        n_mels = 32
+        n_stft = 5
+        mel_spec = torch.randn(2, n_mels, 32) ** 2
+
+        # Single then transform then batch
+        expected = transforms.InverseMelScale(n_stft, n_mels)(mel_spec).repeat(3, 1, 1, 1)
+
+        # Batch then transform
+        computed = transforms.InverseMelScale(n_stft, n_mels)(mel_spec.repeat(3, 1, 1, 1))
+
+        # shape = (3, 2, n_mels, 32)
+        self.assertTrue(computed.shape == expected.shape, (computed.shape, expected.shape))
+
+        # Because InverseMelScale runs SGD on randomly initialized values so they do not yield
```

Comment:
Does setting the random seed fix this, see [here](https://github.com/pytorch/audio/blob/master/test/test_functional.py#L83)?

Reply:
It seems not. I ran multiple of run with different seed value but the P-inf distance was pretty much similar.

|seed| P-inf |
|-----|------|
|57011|0.8953|
|38810|0.9012|
|70091|0.9201|
|80441|0.9746|
|7957|0.9411|
|30379|0.9522|
|32164|0.9534|
|92637|0.9684|
|23494|0.9216|
|94651|0.9352|

Reply:
Interesting, how did you set the seed?

Reply:
I looped the test, changing the seed every time.

```
import random
for _ in range(10):
    seed = random.randint(0, 100000)
    print(seed)
    torch.random.manual_seed(seed)
    # run test.
```

Reply:
For reference, there's a discussion about reproducibility [here](https://pytorch.org/docs/stable/notes/randomness.html). We don't need to worry about this here though. Thanks for the inline comment though :)